plugins/parsers/csv/parser.go

```diff
@@ -23,6 +24,9 @@ import (
 
 type TimeFunc func() time.Time
 
+var replacementByte = "\ufffd"
+var commaByte = "\u002C"
```

Comment:
Can you please make this
```suggestion
const replacementByte = "\ufffd"
const commaByte = "\u002C"
```